keras/engine/saving.py

```diff
@@ -491,7 +492,12 @@ def preprocess_weights_for_loading(layer, weights,
     # Returns
         A list of weights values (Numpy arrays).
     """
-    if layer.__class__.__name__ == 'Bidirectional':
+    def convert_nested_bidirectional(weights):
+        """
+        Converts layers nested in `Bidirectional` wrapper by `preprocess_weights_for_loading()`.
+        :param weights: list of of numpy arrays
+        :return: converted list of of numpy arrays
+        """
```

Comment:
Please format the docstring like other docstrings in the codebase.

Reply:
Thanks.